tools/POTools/Services/Extracting/ResourceTextExtractor.cs

```diff
@@ -14,6 +15,11 @@ public IEnumerable<LocalizableTextInfo> Extract(string content, CancellationToke
         using (var reader = new StringReader(content))
             resourceReader = new ResXFileReader(reader);
 
-        return resourceReader.Select(kvp => new LocalizableTextInfo { Id = kvp.Value, Comment = kvp.Key });
+        // Select name, value, and comment from the resource reader.
+        return resourceReader.Select(item => new LocalizableTextInfo {
+            Id = item.value.Replace("\n", Environment.NewLine),
+            ExtractedComment = $"{(item.comment.Length > 0 ? item.comment + " " : string.Empty)}[{item.name}]",
+            TranslatorComment = string.Empty,
```

Comment:
Could you explain what's the point in introducing `TranslatorComment` and setting it to a constant value (empty string)?

Reply:
I was initially confused as to the difference between 'extracted comments' and 'translator comments' (and it seems other software developers also are, as that was what caused my to misunderstand things. Then when the dust settled I realized I had no need for both, but decided to keep them there to keep things clear, even if strictly speaking not necessary

Reply:
If so, then we'd better remove `TranslatorComment`. I don't think it adds any value at this point.

But, of course, I don't mind, if we keep the `ExtractedComment` name for clarity.